epub33/reports/index.html

```diff
@@ -48,44 +53,67 @@
       <h1 id="reports-per-criteria">Reports per documents and criteria</h1>
 
       <dl>
-        <dt><a href="exit_criteria.html#exit-criteria-core">EPUB 3.3 (Core)</a></dt>
+        <dt><a href="exit_criteria.html#exit-criteria-core">EPUB 3.3 (Core)</a> [[epub-33]]</dt>
         <dd>
           <ul>
             <li>
-              <a href="exit_criteria.html#exit-criteria-core-1">Category 1</a>: 
-              see the <a href="https://w3c.github.io/epub-tests/results">EPUB 3.3 Test Results</a>
+              <p>
+                <a href="exit_criteria.html#exit-criteria-core-1">Category 1</a>: each <em>required</em> feature define by [[epub-33]], that has a direct effect on the reading system behavior, and whose behavioral details are mostly specified by [[epub-rs-33]], must have at least two, mutually independent implementations.
+              </p>
+              <p>See the <a href="https://w3c.github.io/epub-tests/results">EPUB 3.3 Test Results</a> document.</p>
             </li>
             <li>
-              <a href="exit_criteria.html#exit-criteria-core-2">Category 2</a>: 
-              see the <a href="https://w3c.github.io/epub-structural-tests/">EPUB 3.3 Structural Test Results</a>
+              <p>
+                <a href="exit_criteria.html#exit-criteria-core-2-3">Categories 2 and 3</a>: each <em>required</em> constraint on EPUB-specific files, as well as restrictions on [=publication resources=], must be enforceable when checking validity through <a href="https://github.com/w3c/epubcheck/releases/tag/v5.0.0">EPUBCheck 5</a>.
+              </p> 
+              <p>See the <a href="https://w3c.github.io/epub-structural-tests/">EPUB 3.3 Structural Test Results</a> document.</p>
             </li>
             <li>
-              <a href="exit_criteria.html#exit-criteria-core-3">Category 3</a>: 
-              see the <a href="epub-properties-use">EPUB 3.3 Metadata Usage Report</a>
+              <p>
+                <a href="exit_criteria.html#exit-criteria-core-4">Category 4</a>: each metadata item defined and required in the [=package document=] defined by [[epub-33]] has sufficient usage by the target communities, i.e., at least two organizations regularly include the metadata in the package document (for publishers) or use them in bookshelves, content rendering, etc. (for reading system developers).
+              </p>
+              <p>
+                See the <a href="epub-properties-use">EPUB 3.3 Metadata Usage Report</a> document.
```

Comment:
Some of these URLs won't resolve. I'm assuming since this PR doesn't change the reports folder structure, the files are remaining as-is. Will open changes for the ones that don't look right.
```suggestion
                See the <a href="epub-properties-use.html">EPUB 3.3 Metadata Usage Report</a> document.
```

Reply:
Actually, `https://w3c.github.io/epub-specs/epub33/reports/epub-properties-use` works properly, and the folder structure does not change. Ie, there references will resolve to the HTML version automatically when done through HTTP (I tested these on my machine via a local server on localhost)

But, who knows... for reviewers it may be more agreeable to directly use the html. So let us go for it...

Reply:
> there references will resolve to the HTML version automatically when done through HTTP

Weird, I did the same thing (checked out the branch and opened on localhost in IIS) and got 404s. It should try to resolve to an index.html file in a directory with the trailing name (does you server do any other kinds of automatic resolution?). But with the extension there's no danger of breakage, so all good.

Reply:
To be honest, I do not remember if I did something special in the local apache setup, I created the config file about 10 years ago... :-)